src/io/hdf5/HDF5RecordingData.hpp

```diff
@@ -87,7 +87,6 @@ class HDF5RecordingData : public AQNWB::IO::BaseRecordingData
    */
   Status writeDataBlockHelper(const std::vector<SizeType>& dataShape,
                               const std::vector<SizeType>& positionOffset,
```

Comment:
Please also remove "* @param type The data type of the elements in the data block." from the docstring